src/styles/auto-prefix.js

```diff
@@ -20,18 +20,29 @@ export default {
     let prefixer = prefixers[userAgent];
     // None found, create a new instance
     if (!prefixer) {
-      prefixer = new InlineStylePrefixer(userAgent);
+      prefixer = new InlineStylePrefixer({userAgent: userAgent});
```

Comment:
This can be debatable. since a careless rename can break the prefixer but with explicit property naming this cannot happen. but that's very rare. so, I don't know I'm ok either way :+1: